mpas_analysis/ocean/sose_transects.py

```diff
@@ -148,7 +148,7 @@ def __init__(self, config, mpasClimatologyTask, refConfig=None):
                         refFieldName = fields[fieldName]['mpas']
 
                     fieldNameUpper = fieldName[0].upper() + fieldName[1:]
-                    fieldNameInTytle = r'{} at {}$^\circ$ Longitude'.format(
+                    fieldNameInTytle = r'{} at {}$^\circ$Longitude'.format(
```

Comment:
I think the space was correct here.

Reply:
I definitely see what you mean.  I agree it's a problem.  Let's see if we can find another way to do a degree sign that's not an o and that doesn't leave weird extra space.  I'll hunt around and get back to you.

Reply:
Following this suggestion, maybe you could just try `$\degree$`?
https://stackoverflow.com/a/46488216/7728169

Reply:
If that doesn't work, the next thing to try might be`'{} at {}\u00B0 Longitude'`:
https://stackoverflow.com/a/8442167/7728169

Reply:
OK, I will try these.  Thanks.